transitions/registration.js

```diff
@@ -265,6 +265,7 @@ module.exports = {
             if (err) {
                 callback(err);
             } else if (registrations.length) { // id collision, retry
+                logger.warn('Registration ID ' + id + ' is not unique, retrying...');
```

Comment:
I actually used `warn` because this can create issues with sentinel and should be infrequent.  If it starts happening a lot it's highly recommended to increase the id format/length because the transitions are processed serially and this could potentially loop forever.  We actually had a case where sentinel wasn't doing anything else except for stuck in that loop.